benchmarks/scripts/cache_cleaning_workers_post_query.py

```diff
@@ -0,0 +1,135 @@
+import mysql.connector
+from mysql.connector import Error
+import json
+import paramiko
+import argparse
+
+def create_connection(host_name, user_name, user_password, db_name):
```

Comment:
@ethanyzhang how does pbench manage the utility functions like create_connection? It also appears in benchmarks/scripts/cache_cleaning_coordinator_post_query.py

Reply:
@agrawalreetika I think it may be better to move `create_connection` to a different file then include it in both the worker script and the coordinator script.

Say this file is called `presto_utils.py`, then here you can do:
```python
from presto_utils import create_connection
```